Dockerfile

```diff
@@ -0,0 +1,32 @@
+# ---- Base Node ----
+FROM node:10-stretch AS base
```

Comment:
@dmlb2000 After doing some additional testing, we should ok to move to alpine based images as it will significantly reduce image size.